### PR TITLE
Update Sensefai URL

### PR DESCRIPTION
**Pull request by Mr-HoDL58**

### CODE REVIEW

File: website/index.html [hunk 1]
```diff
@@ -103,7 +103,7 @@ <h3>{% t dict.home.partners %}</h3>
     </div>
     <div class="row p-4">
       <div class="col-md-4 align-self-center">
-        <a href="https://sensifia.com" target="_blank">
+        <a href="https://sensifai.com" target="_blank">
           <img alt="Sensifia" src="{{ site.url }}/assets/images/partners/sensifia.png" />
```
[the remark this answers is not in the document]
Comment (Mr-HoDL58) [reply website/index.html:107]: Yes mo is going to obtain new logos for sensefia and ip minter.

[the remark this answers is not in the document]
Comment (Mr-HoDL58) [reply website/index.html:107]: We just update url at this point so its correct. Do you agree?

[the remark this answers is not in the document]
Comment (Ja7ad) [reply website/index.html:107]: > Yes mo is going to obtain new logos for sensefia and ip minter.

Just sensifai:

https://sensifai.com/wp-content/uploads/2019/05/logo.svg

[the remark this answers is not in the document]
Comment (Ja7ad) [reply website/index.html:107]: > We just update url at this point so its correct. Do you agree?
> 

Yes, url correct

[the remark this answers is not in the document]
Comment (Mr-HoDL58) [reply website/index.html:107]: any chance you can get IP minters svg. We are going to use dual images for light and dark.

[the remark this answers is not in the document]
Comment (Mr-HoDL58) [reply website/index.html:107]: also for some reason it will not download this file. Can you post the image as a file please.

**Pull request merged**